governance/understanding.rst

```diff
@@ -234,13 +233,13 @@ roles.
 Voting on proposals is updated in real time via P2P messages and stored
 by Dash in cache files, so current winning proposals and the total
 allocation of the available budget are always open and visible to
-everyone. `Dash Nexus <https://app.dashnexus.org/>`__ is a popular site used
+everyone. `Dash Central <https://www.dashcentral.org/>`__ is a popular site used
```

Comment:
Last change :slightly_smiling_face: This will get users directly to the proposal status page.
```suggestion
everyone. `Dash Central <https://www.dashcentral.org/budget/>`__ is a popular site used
```

Reply:
added